### ISSUE TITLE
Connection failed if WiFi is on on Garmin device

### ISSUE THREAD

**Issue by floschl** [gimportexportdevs/gexporter#16]
Thanks a lot for this great plugin.
I got it working with my Pixel 5 smartphone and my Edge 530.

I got it working if I disable the WiFi on my Edge 530, otherwise it throws a "Connection failed" error.
Why is this the case and is there a workaround? It is a bit cumbersome to always disable/enable Wifi if I want to download a gpx file from my smartphone.

Thanks a lot!

**Comment by haraldh** [comment]
Otherwise it tries to connect to the internet and not to the phone. Sorry, a workaround I should document for those devices, which can connect to the internet themselves.

**Comment by floschl** [comment]
So this means that there is no other workaround than disabling/enabling WiFi if I want now and then sync a GPX from my phone?

**Comment by haraldh** [comment]
yes, sadly not due to garmins sucky public programming interface for us developers
